codeHF/runtest.sh

```diff
@@ -95,8 +97,10 @@ function MsgErr { echo -e "\e[1;31m$@\e[0m"; }
 # Handle dependencies. (latest first)
 [ $DOO2_TASK_D0 -eq 1 ] && { DOO2_SEL_D0=1; }
 [ $DOO2_SEL_D0 -eq 1 ] && { DOO2_CAND_2PRONG=1; DOO2_PID_TPC=1; DOO2_PID_TOF=1; }
+[ $DOO2_SEL_Lc -eq 1 ] && { DOO2_CAND_3PRONG=1; DOO2_PID_TPC=1; DOO2_PID_TOF=1; }
 [ $DOO2_CAND_2PRONG -eq 1 ] && { DOO2_SKIM=1; }
 [ $DOO2_TASK_DPLUS -eq 1 ] && { DOO2_CAND_3PRONG=1; }
+[ $DOO2_TASK_Lc -eq 1 ] && { DOO2_SEL_Lc=1; }
```

Comment:
The order seems wrong as I believe that Lc task requires the Lc selector to run first.
Also put those two lines one after the other for clarity (both after `DPLUS`).